### ISSUE TITLE
Right click an empty decon planner, crash!

### ISSUE THREAD

**Issue by Omnifarious** [dewiniaid/BlueprintExtensions#5]
Just as the title says. Create a new decon planner by clicking the red icon in the blueprint book window. Right click on it. You get same strange error message from BlueprintExtensions and your game crashes.

**Comment by Omnifarious** [comment]
I am now also unable to reproduce the issue. :-/ How strange. I'll close this, and remember to capture a screenshot next time it happens, as well as a listing of my mods directory and a copy of my mod-list.json file.

**Comment by Omnifarious** [comment]
I maanged to re-create it. It may be some sort of incompatibility with the Black Market mod.

https://steamcommunity.com/sharedfiles/filedetails/?id=1541717639

This time it happened when I clicked on a GUI element in the Black Market mod, not when I clicked on a decon planner.

I don't think it has anything to do with the Black Market mod though, given the message. It seems like a failure to set up a local variable that the click handler is expecting.
